website/docs/documentationfile.md

```diff
@@ -0,0 +1,46 @@
+Enables C# xmlDocumentationFile
+
+The `xmlDocumentationFile` option is used to include [XML comments](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/) in a DLL that has been included in a .NET framework or another C# project. These XML comments can then be referenced by other projects when placed alongside the corresponding DLL.
+
+This feature sets the [documentationfile](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/output#documentationfile) option in a C# project's .csproj file for each respective [configuration](https://premake.github.io/docs/configurations/)
+
+## Usage ##
+To use xmlDocumentationFile, add the following line to your project configuration in your premake script:
+
+```lua
+documentationfile "targetdir"
+```
+### Parameters ###
+`targetdir` is the directory where the documentation file should be placed after building the project using visual studio.
+
+### Examples ###
+
+When you specify an empty string for `documentationfile`, the following filepath will be generated:
+```%{targetdir}/%{prj.name}.xml```
+```lua
+documentationfile ""
+```
+If you specify a custom target directory like this:
+```lua
+documentationfile "%{prj.location}/bin/test"
+```
+ the following filepath will be generated:
+```bin\test\%{prj.name}.xml```
+### Applies To ###
+
+Project configurations.
+
+### Availability ###
+
+Premake 5.0 or later.
```

Comment:
Not sure which beta we are currently to be more accurate.

Reply:
Should be beta3, so `Premake 5.0 beta3 or later.`.

Reply:
will change that